client/src/components/FeedbackForm/FeedbackForm.js

```diff
@@ -62,8 +62,6 @@ const FeedbackForm = () => {
         return (
           <Form styleName="container">
             <FormGroup>
-              <FormField name="name" label="Full Name" required disabled />
```

Comment:
With this removal, we no longer need lines 35-38 of the file. Also that `useMemo` could just be an object now.

https://github.com/TACC/Frontera-Portal/pull/319/files#diff-c492eda7296e3435975a59882ed51534ac3727a583bbe479954b4210aaeea6aaR35-R38

Reply:
cool. i'll make that change and add them back in the `onSubmit`/`TICKET_CREATE`

Reply:
@rstijerina , Updated.  Can you take a look at the latest commit? Thanks!

Reply:
@rstijerina , made the additional refactors.  Can you check the backend/frontend changes in the recent commits.